bundle/src/main/java/com/jfrog/ide/eclipse/configuration/PreferenceConstants.java

```diff
@@ -17,4 +17,7 @@ public class PreferenceConstants {
 	public static final String GRADLE_PLUGIN_QUALIFIER = "org.eclipse.buildship.core";
 	public static final String GRADLE_DISTRIBUTION = "gradle.distribution";
 	
+	// Default exluded paths pattern
+	public static final String DEFAULT_EXCLUSIONS = "**/*{.git,test,node_modules,target}*";
```

Comment:
can we add a way to configure this attribute?

Reply:
will be removed for now, and on the next version will be implemented